client/public/styles/global.css

```diff
@@ -0,0 +1 @@
+@import url('https://fonts.googleapis.com/css2?family=Source+Code+Pro:wght@600&display=swap');
```

Comment:
I would prefer if we hosted the font locally for speed, and also because I don't like google.